spec/models/post_spec.rb

```diff
@@ -0,0 +1,55 @@
+require 'rails_helper'
+
+RSpec.describe Post, type: :model do
+  let(:first_user) { User.create(name: 'Tom', photo: 'https://unsplash.com/photos/F_-0BxGuVvo', bio: 'Teacher from Mexico.', posts_counter: 5) }
+  subject do
+    described_class.new(author: first_user, title: 'Hello world', text: 'This is my first post', comments_counter: 0,
+                        likes_counter: 0)
+  end
+
+  before { subject.save }
+
+  # Validation tests
+  describe 'Validations' do
```

Comment:
Great job @anshuman7negi 🤝

- [ ] As stated in the project requirements, kindly ensure the `post` model has validation tests where the title must not exceed 250 characters.